bears/markdown/MarkdownBear.py

```diff
@@ -9,9 +9,14 @@
         diff_message='The text does not comply to the set style.')
 class MarkdownBear:
     """
-    Raises issues against style violations on markdown files.
+    Check and correct Markdown style violations automatically.
+
+    See <https://github.com/wooorm/remark-lint> for details about the tool
+    below.
     """
 
+    LANGUAGES = ("MarkDown",)
```

Comment:
Thats a weird way of writing "markdown"
Maybe we should have a list of acceptable languages which is checked somewhere to avoid errors like "CPP" instead of "C++" etc